blog/26-08-2022-creating-a-hook-for-the-go-sdk.mdx

```diff
@@ -0,0 +1,148 @@
+---
+slug: creating-a-hook-for-the-go-sdk
+title: Creating a Hook for the golang-sdk
+authors: [skyerus]
+tags: [golang, blog, tutorial, golang-sdk, hooks]
+unlisted: false
+---
+
+## Hooks
+
+A [Hook](./../docs/specification/sections/hooks) taps into one or more of the flag evaluation's
```

Comment:
Apparently you can't do relative paths in blog posts.. bummer.

Reply:
Ah that's annoying, it worked locally :(

Reply:
Will revert